sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
nit: Add license info at the top of file

<!--
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License. See License.txt in the project root for
 * license information.
-->

Reply:
Fixed.